hawc/apps/assessment/templates/assessment/assessment_downloads.html

```diff
@@ -184,6 +184,90 @@ <h3 class="card-header">Ecological</h3>
     </div>
   </div>
 </div>
+
+
+<p class='form-text text-muted mb-3'>
+  Assessment downloads availalbe for team members or higher.
+</p>
+<div class="row">
+<div class="col-xl-6">
+    <h3>Data extractions</h3>
+    <div class="bg-light border shadow p-3">
+        <p><b>Animal bioassy export</b></p>
+        {% url 'animal:api:assessment-full-export' assessment.pk as the_url %}
```

Comment:
Can you make a component for this little collection of spans and links?

```
{% include 'assessment/components/foo.html' with url=the_url %}
```

Something like that, but with a better name :)